firmware_mod/www/cgi-bin/action.cgi

```diff
@@ -121,45 +127,45 @@ if [ -n "$F_cmd" ]; then
        /system/sdcard/controlscripts/rtsp-h264 stop
     ;;
     settz)
+       ntp_srv=$(printf '%b' "${F_ntp_srv//%/\\x}")
 
-	ntp_srv=$(printf '%b' "${F_ntp_srv//%/\\x}")
-
-	#lecture fichier ntp_serv.conf
-	conf_ntp_srv=$(cat /system/sdcard/config/ntp_srv.conf)
+       #lecture fichier ntp_serv.conf
+       conf_ntp_srv=$(cat /system/sdcard/config/ntp_srv.conf)
 
-    if [ $conf_ntp_srv != "$ntp_srv" ]; then
-    echo "Setting NTP Server to '$ntp_srv'...<br/>"
-    echo "$ntp_srv" > /system/sdcard/config/ntp_srv.conf
-    echo "Syncing time on '$ntp_srv'...<br/>"
+      if [ $conf_ntp_srv != "$ntp_srv" ]; then
+        echo "<p>Setting NTP Server to '$ntp_srv'...</p>"
+        echo "$ntp_srv" > /system/sdcard/config/ntp_srv.conf
+        echo "<p>Syncing time on '$ntp_srv'...</p>"
         if [ "$(/system/sdcard/bin/busybox ntpd -q -n -p $ntp_srv 2>&1)" -eq 0 ]; then
-          echo "<br/>Success<br/>"
-        else echo "<br/>Failed<br/>"
-		fi
-    fi
+          echo "<p>Success</p>"
+        else
+          echo "<p>Failed</p>"
+        fi
+      fi
 
-	#lecture fichier ntp_serv.conf
-	conf_ntp_srv=$(cat /system/sdcard/config/ntp_srv.conf)
+      #lecture fichier ntp_serv.conf
+      conf_ntp_srv=$(cat /system/sdcard/config/ntp_srv.conf)
 
-	tz=$(printf '%b' "${F_tz//%/\\x}")
+      tz=$(printf '%b' "${F_tz//%/\\x}")
       if [ "$(cat /etc/TZ)" != "$tz" ]; then
-        echo "Setting TZ to '$tz'...<br/>"
+        echo "<p>Setting TZ to '$tz'...</p>"
         echo "$tz" > /etc/TZ
-        echo "Syncing time...<br/>"
-        if [ "$(/system/sdcard/bin/busybox ntpd -q -n -p $conf_ntp_srv 2>&1)" -eq 0 ]; then
-          echo "<br/>Success<br/>"
-        else echo "<br/>Failed<br/>"
+        echo "<p>Syncing time...</p>"
+        if [ "$(/system/sdcard/bin/busybox ntpd -q -n -p time.google.com 2>&1)" -eq 0 ]; then
```

Comment:
this may break ntp configuration via file?

Reply:
@jmtatsch Fixed. Thanks for the catch. Turns out that I missed out on the ntp form in the ``status.cgi`` as well.